src/libstore/build/local-derivation-goal.cc

```diff
@@ -1777,7 +1777,7 @@ void LocalDerivationGoal::runChild()
                     if (pathExists(path))
                         ss.push_back(path);
 
-                dirsInChroot.emplace(settings.caFile, "/etc/ssl/certs/ca-certificates.crt");
+                dirsInChroot.try_emplace("/etc/ssl/certs/ca-certificates.crt", settings.caFile, true);
```

Comment:
I know it doesn't change behavior, but I would still do
```suggestion
                if (settings.caFile != "")
                    dirsInChroot.try_emplace("/etc/ssl/certs/ca-certificates.crt", settings.caFile, true);
```
Setting a non-existent file is a dubious thing that should get a log message. Setting the empty string is the preferred way to not set any SSL path. Even once a `std::optional` refactor happens, an empty string would still be the surface syntax for `nix.confg` and the env vars.